docs/user/introduction.md

```diff
@@ -1,4 +1,3 @@
-Introduction
-------------
+This section provides an in depth guide on the usage of MasterMind, showcasing how to create projects, assign user roles, register clusters, register or deploy services, and register NGSI subscriptions.
 
-TODO
+MasterMind is made out of two main components: the UI, and the API. The average user will only need to interact with the UI to make use of most of MasterMind's features, but MasterMind can be used via API calls only if needed. The manual will cover both UI usage as well as the API.
```

Comment:
MasterMind is made out of two main components: the UI, and the API. The average user only needs to interact with the UI to make use of most of MasterMind's features, but MasterMind can be used via API calls if needed. The manual covers both UI usage as well as the API.